middleware/proxy_1_11.go

```diff
@@ -0,0 +1,27 @@
+// +build go1.11
+
+package middleware
+
+import (
+	"fmt"
+	"github.com/labstack/echo"
+	"net/http"
+	"net/http/httputil"
+)
+
+func proxyHTTP(t *ProxyTarget, c echo.Context, config ProxyConfig) http.Handler {
+	proxy := httputil.NewSingleHostReverseProxy(t.URL)
+	proxy.ErrorHandler = func(resp http.ResponseWriter, req *http.Request, err error) {
+		descr := t.URL.String()
+		if len(t.Name) > 0 {
+			descr = fmt.Sprintf("%s(%s)", t.Name, t.URL.String())
+		}
+		c.Logger().Warnf("remote %s unreachable, could not forward: %v", descr, err)
+		c.Error(echo.NewHTTPError(
```

Comment:
Perhaps you can add an entry here https://github.com/labstack/echo/blob/master/echo.go#L252 and use it.